### ISSUE TITLE
RTX 3060 LHR v1 locks when i turn a game on

### ISSUE THREAD

**Issue by ceoforex1** [trexminer/T-Rex#1306]
It locks on 750 mV (85 watts) as shown on msi after burner if i mine while I game. this weas not a problem in 80% unlocks. It keeps locked even if i turn game off and restart mining. im having to restart windows

**Comment by zulu1905** [comment]
playing game and sametime mining !!! it is not healthy

**Comment by rafaelgirotti** [comment]
Yeah I have the same problem but not just with a game or video. Even only using VS Code the hashrate eventually drops and won't come back until I reboot windows
